inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -1614,6 +1614,14 @@ InferenceEngine::details::CNNLayerCreator::CNNLayerCreator(const std::shared_ptr
         return res;
     });
 
+    addSpecificCreator({"Interpolate"}, [](const std::shared_ptr<::ngraph::Node>& node,
```

Comment:
Why do you need to have a specific creator?

Reply:
I'll remove that

Reply:
So it turns out without the specific creator we're getting IE exceptions, I have added it again.
```
RuntimeError: Interpolate operation has a form that is not supported. Resize should be converted to Interp operation.